apps/web/src/components/common/TextInput/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { cva } from 'class-variance-authority';
+import { forwardRef, InputHTMLAttributes, Ref } from 'react';
+
+import cn from '@/utils/cn';
+
+const inputVariants = cva(
+  'w-full rounded-base border bg-white px-3.5 py-2.5 text-body1 text-main placeholder:text-weak',
+  {
+    variants: {
+      state: {
+        default: 'border-main focus:border-primary',
+        error: 'focus:border-error',
```

Comment:
p3: 
property key에 경우 따로 상수로 정의하여 사용하는 것이 좋아보입니다.

```suggestion
        const KEY = {
          default: "default",
          error: "error"
        }
        
        
       [KEY.default]: 'border-main focus:border-primary',
       [KEY.error]: 'focus:border-error',
       
       
       ...
         const inputState = errorMessage ? KEY.error : KEY.default;

```

Reply:
제안 감사드립니다!!
KEY보다 조금 의미있는 네이밍이면 좋을 것 같아서 `STATE`로 정의했는데 괜찮을까요? 컨벤션 맞추면 좋을거같습니다!
```js
const STATE = {
  default: 'default',
  error: 'error',
} as const;
```